### ISSUE TITLE
[BUG] Blowtorch gamepass does not work

### ISSUE THREAD

**Issue by Axiewu** [Manufacturing-Department/issue-tracker#61]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Location

Live

### Game Version

918

### Expected Behavior

What's happening? After buying one-life blowtorch gamepass, you don't get it. I was also told that kingpins spawn with it so they do not need it to buy it as well.
Expected behaviour? After purchasing the gamepass, you immediately get the blowtorch. 

### Steps To Reproduce

1. Join as a CD // Any server
2. Spawn
3. Buy one-life blowtorch gamepass
4. Didn't get it.

### Anything else?

N/A

**Comment by spenserm10** [comment]
I think they meant to say the "Developer Product" of sorts since you can buy it multiple times. I'm pretty sure you do get the blowtorch with the Kingpin gamepass, but I cannot confirm.

**Comment by FakadoDev** [comment]
> I think they meant to say the "Developer Product" of sorts since you can buy it multiple times. I'm pretty sure you do get the blowtorch with the Kingpin gamepass, but I cannot confirm.

Yeah you get it when you spawn as a kingpin. But it would be awesome if non-kingpins could actually buy it and not ask us to drop.
